src/interface/src/app/services/auth.service.ts

```diff
@@ -133,19 +133,45 @@ export class AuthService {
       );
   }
 
-  changePassword(password1: string, password2: string): Observable<any> {
+  changePassword(currentPassword: string, newPassword1: string, newPassword2: string): Observable<any> {
     return this.http.post(
       this.API_ROOT.concat('password/change/'),
```

Comment:
The path is `/planscape-backend/dj-rest-auth/password/reset/confirm/ `

Reply:
This is existing functionailty that was not modified by me. I hope that this flow has not been modified on the backend as it would break the existing create new user flow.